libraries/midi/src/Midi.cpp

```diff
@@ -16,10 +16,9 @@
 #include <QtCore/QLoggingCategory>
 
 #if defined Q_OS_WIN32
-#include "Windows.h"
-#endif
```

Comment:
Why was this `#endif` removed? This seems to make the entire file a no-op on non-windows systems.

Reply:
The next line below this was an #if defined Q_OS_WIN32, seemed kind of useless added complexity